07-routing-gossip.md

```diff
@@ -829,16 +825,15 @@ The addition of timestamp and checksum fields allow a peer to omit querying for
 
 ### The `gossip_timestamp_filter` Message
 
-1. type: 265 (`gossip_timestamp_filter`) (`gossip_queries`)
+1. type: 265 (`gossip_timestamp_filter`)
 2. data:
     * [`chain_hash`:`chain_hash`]
     * [`u32`:`first_timestamp`]
     * [`u32`:`timestamp_range`]
 
 This message allows a node to constrain future gossip messages to
-a specific range.  A node which wants any gossip messages would have
-to send this, otherwise `gossip_queries` negotiation means no gossip
-messages would be received.
+a specific range.  A node which wants any gossip messages has
+to send this, otherwise no gossip messages would be received.
```

Comment:
Should we also codify the behavior we talked about re "all or nothing" re the timestamp range? So that implementations only ever use this to signal that they want to receive updates, or don't want to receive any updates at all (so no legacy announcement back fill).

Reply:
Seems reasonable to me, but I think it would fit better in a separate PR.

Reply:
This is unrelated to this PR, I agree it should be a separate one.